superbench/benchmarks/micro_benchmarks/cublas_function/CMakeLists.txt

```diff
@@ -0,0 +1,53 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+cmake_minimum_required(VERSION 3.18)
+project(CublasBenchmark)
+
+include(../cuda_common.cmake)
+
+SET(SRC "cublas_benchmark.cu" CACHE STRING "source file")
+SET(TARGET_NAME "cublas_function" CACHE STRING "target name")
+
+if(NOT CMAKE_BUILD_TYPE)
+  set(CMAKE_BUILD_TYPE Release)
+endif()
+
+set(CMAKE_CXX_FLAGS "-Wall -Wextra -std=c++11 -march=native")
+set(CMAKE_CXX_FLAGS_DEBUG "-g")
+set(CMAKE_CXX_FLAGS_RELEASE "-O2")
+
+find_package(CUDA)
```

Comment:
It is no longer necessary to call find_package(CUDA) for compiling CUDA code. Instead, list CUDA among the languages named in the top-level call to the project() command, or call the enable_language() command with CUDA.

reference: https://cmake.org/cmake/help/latest/module/FindCUDA.html

Reply:
![image](https://user-images.githubusercontent.com/37182275/118911745-45404580-b959-11eb-98fa-f0de37bf9967.png)
what extra setting should I add?

Reply:
Please refer https://github.com/microsoft/superbenchmark/blob/main/superbench/benchmarks/micro_benchmarks/kernel_launch_overhead/CMakeLists.txt.
It works.

Reply:
thank you for the comments, I have removed find_package(CUDA) and replace them according to cmake document with new usage